script/room.js

```diff
@@ -797,6 +797,7 @@ var Room = {
 				$SM.set('stores["'+k+'"]', 0);
 			}
 			
+			k = _(k);
```

Comment:
I think I'd prefer to use a different variable here, since `k` is used for object lookups and stuff. If someone moves this line, or adds more lookups below it (thinking it's fine, 'cause we do it above), things will break unexpectedly.